explainers/subresource-loading.md

```diff
@@ -137,27 +120,38 @@ just prevents delays in starting the subresource fetch.
 Suppose that the bundle, `subresources.wbn`, includes the following resources:
 
 ```
-- https://example.com/scope/a.js (which depends on ./b.js)
-- https://example.com/scope/b.js
-- https://example.com/scope/c.png
+- https://example.com/dir/a.js (which depends on ./b.js)
+- https://example.com/dir/b.js
+- https://example.com/dir/c.png
+- urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6
 - … (omitted)
 ```
 
 ### The main document
 
 ```html
 <link rel="webbundle"
-    href="https://example.com/subresources.wbn"
-    scope="https://example.com/scope/"
+  href="https://example.com/dir/subresources.wbn"
+  resources="https://example.com/dir/a.js \
+             https://example.com/dir/b.js \
+             https://example.com/dir/c.png \
```

Comment:
We wouldn't write `\`s in the HTML source:
```suggestion
  resources="https://example.com/dir/a.js
             https://example.com/dir/b.js
             https://example.com/dir/c.png
```

Reply:
Done.